tests/test_helpers.py

```diff
@@ -28,6 +30,11 @@
 )
 from anndata.utils import axis_len
 
+jax.config.update(
+    "jax_enable_x64",
+    True,  # noqa: FBT003
+)  # Explicit conversion to JAX float64
```

Comment:
don’t call code on top-level.

instead, call it in fixture setups instead and undo what you did in fixture teardown:

```py
@pytest.fixture
def _give_me_a_name() -> None:
	# Explicit conversion to JAX float64
	jax.config.update("jax_enable_x64", True)  # noqa: FBT003
	yield
	# TODO: undo what you did above
```